lib/flight_job/cli.rb

```diff
@@ -90,14 +90,38 @@ def self.create_command(name, args_str = '')
       c.summary = 'List your rendered scripts'
     end
 
+    create_command 'view-script', 'SCRIPT_ID' do |c|
+      c.summary = 'View the content of a script'
+    end
+
     create_command 'info-script', 'SCRIPT_ID' do |c|
       c.summary = 'Display details about a rendered script'
     end
 
-    # XXX: Consider making the method signature: TEMPLATE_NAME [SCRIPT_NAME]
-    create_command 'create-script', 'TEMPLATE_NAME' do |c|
+    create_command 'create-script', 'TEMPLATE_NAME [SCRIPT_NAME]' do |c|
       c.summary = 'Render a new script from a template'
-      c.slop.bool '--stdin', 'Provide the answers via STDIN as JSON'
+      c.slop.string '--answers', <<~MSG.chomp, meta: 'JSON|@filepath|@-'
+        Provide the answers as a JSON string.
+        Alternatively specify a file containing the answers with @filepath or STDIN as @-
+      MSG
+      c.slop.string '--notes', <<~MSG.chomp, meta: 'NOTES|@filepath|@-'
+        Provide additional information about the script
+        Alternatively specify a file containing the notes with @filepath or STDIN as @-
```

Comment:
The webapp is going to assume that the notes are markdown.  We should mention something about markdown format being supported.  Perhaps:

```
Provide additional information about the script (Markdown formatting is supported).

...
```

Reply:
c2968f93078c9098653107f48c025a1bf2112d29